### PR TITLE
[ecotouch] Add divider for analog values and additional temperature values (#5165)

### LINKED ISSUE #5165: [ecotouch] Readable Tags missing values from Air-Water Heatpump
The current implementation seems to be build for a geothermal heatpump only. Some values from air-water heatpump (like EcoTouch Ai1 Air) are missing.

## Possible Solution
Will add additional tags to EcoTouchTags. E.g. I2020 for surrounding temperature T2.

### PR DESCRIPTION
**Pull request by Digitalvitamin**
Also documented new values and some state mappings.

Closes #5165

Signed-off-by: Christian Stübner <c.stuebner@digitalvitamin.de> (github: Digitalvitamin)

### CODE REVIEW

File: bundles/binding/org.openhab.binding.ecotouch/README.md [hunk 1]
```diff
@@ -123,6 +126,28 @@ where `<eventType>` is one of the following values:
 | Handschaltung Vierwegeventil | manual_4wayvalve |  | yes | (H-0-A)
 | Handschaltung Multiausgang Ext. | manual_multiext |  | yes | (H-0-A)
 
+### Mappings
+
+Heatpump event type "state" can be mapped to strings. Different heatpump models seem to have slightly different state values.
+Mappings for the Ai1 Geo have been coppied from [here](http://www.haustechnikdialog.de/Forum/t/173357/Waterkotte-via-Ethernet-OpenHAB-und-Android-App?page=6).
```
Comment (9037568) [inline bundles/binding/org.openhab.binding.ecotouch/README.md]: "copied"

**Comment by Digitalvitamin** [comment]
Thanks for the review. Fixed the typo.
I did not get any existing users to test the changes though. How would I find/know any?

**Comment by 9037568** [comment]
You could post a thread in the [community forum.](http://community.openhab.org/)

**Comment by Digitalvitamin** [comment]
Opened a new [topic](https://community.openhab.org/t/ecotouch-binding-update-users-wanted-for-testing-the-new-version/28218).

**Comment by Digitalvitamin** [comment]
The binding still work for [existing users](https://community.openhab.org/t/ecotouch-binding-update-users-wanted-for-testing-the-new-version/28218/6?u=digitalvitamin).
From my point of view its ready to merge.

**Comment by 9037568** [comment]
I'd like to give the feedback thread a couple more days soak time, then I think we're good to merge.

**Comment by 9037568** [comment]
Thanks, @Digitalvitamin !

**Pull request merged**